website/static/js/contribRemover.js

```diff
@@ -0,0 +1,312 @@
+/**
+ * Controller for the Remove Contributor modal.
+ */
+'use strict';
+
+var $ = require('jquery');
+var ko = require('knockout');
+var bootbox = require('bootbox');
+var Raven = require('raven-js');
+
+var oop = require('./oop');
+var $osf = require('./osfHelpers');
+var Paginator = require('./paginator');
+
+function getNodesOriginal(nodeTree, nodesOriginal) {
```

Comment:
I would prefer to see a single `getNodesOriginal` function that is shared between this file and `nodesPrivacy.js`, rather than having a duplicate function. Let's stay [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)! It's a bit of a junk drawer, but I guess [`osfHelpers.js`](https://github.com/CenterForOpenScience/osf.io/blob/336d249199693b5073842d97f74d12ca74ac496b/website/static/js/osfHelpers.js) would be the best place for it?